src/components/menu/nz-submenu.component.ts

```diff
@@ -144,7 +145,7 @@ export class NzSubMenuComponent implements OnInit, OnDestroy, AfterViewInit {
   }
 
   ngOnInit() {
-    this._$mouseSubject.debounceTime(300).subscribe((data: boolean) => {
+    debounceTime.call(this._$mouseSubject, 300).subscribe((data: boolean) => {
```

Comment:
That could be done in a separate request, since this is only for fixing the rxjs imports, shouldn't change any other part of the lib.